tests/components/plex/test_init.py

```diff
@@ -106,10 +118,13 @@ async def test_setup_with_photo_session(hass, entry, mock_websocket, setup_plex_
     assert sensor.state == "0"
 
 
-async def test_setup_when_certificate_changed(hass, entry):
+async def test_setup_when_certificate_changed(hass, requests_mock):
     """Test setup component when the Plex certificate has changed."""
+    await async_setup_component(hass, "persistent_notification", {})
+    requests_mock.get("https://plex.tv/users/account", text=PLEXTV_ACCOUNT_PAYLOAD)
+    requests_mock.get("https://plex.tv/api/resources", text=PLEXTV_RESOURCES)
 
-    old_domain = "1-2-3-4.1234567890abcdef1234567890abcdef.plex.direct"
+    old_domain = "1-2-3-4.1111111111ffffff1111111111ffffff.plex.direct"
```

Comment:
Why did this change? Not that it shouldn't, just having a hard time following so want to make sure I understand.

Reply:
Just to have a different value that wasn't so close to the "correct" one. Easier to visually identify.